transport/lwm2m/src/main/resources/tb-lwm2m-transport.yml

```diff
@@ -185,7 +185,7 @@ transport:
     report_timeout: "${TB_TRANSPORT_SESSIONS_REPORT_TIMEOUT:30000}"
   json:
     # Cast String data types to Numeric if possible when processing Telemetry/Attributes JSON
-    type_cast_enabled: "${JSON_TYPE_CAST_ENABLED:true}"
+    type_cast_enabled: "${JSON_TYPE_CAST_ENABLED:false}"
```

Comment:
You no longer need this change because you don't use the json converter anymore. Please keep as-is.